tests/pdk/finfet_pdk/test_routing.py

```diff
@@ -30,7 +30,7 @@ def test_cmp_clk():
             ["xdp"],
             ["mn0"]
             ]},
-        {"constraint": "AspectRatio", "subcircuit": "ckt_comparator", "ratio_low": 0.5, "ratio_high": 2}
+        {"constraint": "AspectRatio", "ratio_low": 0.5, "ratio_high": 2}
```

Comment:
Other "subcircuit" constraint does not have "subcircuit"

Reply:
Yes, this redundant attribute is recently retired.